### PR TITLE
docs/managed-victoriametrics: add notifications setup

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Added documentations how to setup notifications into managed-victoriametrics

### CODE REVIEW

File: docs/managed-victoriametrics/setup-notifications.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+weight: 6
+title: Notifications in Managed VictoriaMetrics
+menu:
+  docs:
+    parent: "managed"
+    weight: 6
+---
+# Notifications in Managed VictoriaMetrics
+
+This section add ability to setup channels for sending alerting notifications.
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:12]: The guide covers how to enable email and Slack notifications

File: docs/managed-victoriametrics/setup-notifications.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+The document covers the following topics
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:14]: Table of Content:

File: docs/managed-victoriametrics/setup-notifications.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+1. [Setup slack channels](#setup-slack-channels)
+2. [Setup emails](#setup-emails)
+3. [Send test notification](#send-test-notification)
+
+When you entered to the notification section you will be able to fill in the channels in which you
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:19]: why do we need this sentence and screen below?

File: docs/managed-victoriametrics/setup-notifications.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+3. [Send test notification](#send-test-notification)
+
+When you entered to the notification section you will be able to fill in the channels in which you 
+want to receive notifications
+
+<p>
+  <img src="notifications-start.png" width="800">
+</p>
+
+## Setup slack channels
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:26]: slack -> Slack

File: docs/managed-victoriametrics/setup-notifications.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+To send alerts to Slack, it is required to set up a webhook. How to do this is indicated on the following 
+link <a href="https://api.slack.com/messaging/webhooks" target="_blank">https://api.slack.com/messaging/webhooks</a>
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:29]: don't use <a> - use markdown for links

File: docs/managed-victoriametrics/setup-notifications.md (hunk 1 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/setup-notifications.md]: updated

File: docs/managed-victoriametrics/setup-notifications.md (hunk 2 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/setup-notifications.md]: updated

File: docs/managed-victoriametrics/setup-notifications.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    parent: "managed"
+    weight: 6
+---
+# Notifications in Managed VictoriaMetrics
+
+The guide covers how to enable email and Slack notifications.
+
+Table of content:
+1. [Setup slack channels](#setup-slack-channels)
+2. [Setup emails](#setup-emails)
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:16]: setup email notifications

File: docs/managed-victoriametrics/setup-notifications.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+To send alerts to Slack, it is required to set up a webhook. How to do this is indicated on the following 
+link <a href="https://api.slack.com/messaging/webhooks" target="_blank">https://api.slack.com/messaging/webhooks</a>
+
+When webhook enabled correctly the next step is fill channels and webhook fields in the managed-victoriametrics.
+
+<p>
+  <img src="notifications-slack-channels-webhook.png" width="800">
+</p>
+
+If you want to set slack channels just enter one or more channels into input and press enter or choose it after each input.
```
Comment (tenmozes) [inline docs/managed-victoriametrics/setup-notifications.md:37]: remove all `just`

**Review by tenmozes — CHANGES_REQUESTED** [review]

**Review by tenmozes — APPROVED** [review]

**Pull request merged**